daq/forchestrator.py

```diff
@@ -6,6 +6,7 @@
 import faucet_event_client
 import http_server
 from faucet_states_collector import FaucetStatesCollector
+from local_states_collector import LocalStatesCollector
```

Comment:
I know it's annoying, but can you rename this local_state_collector (both module and class).  We should do faucet_state_collector too but that should be a separate PR.

Reply:
Sure. I had the same idea but was just thinking there would be too much to modify. I will go ahead modify it.

Reply:
It's always a trade-off.  Usually I go for it if it's "in" the code that I'm working with. In this case I think it was reasonable... not to much churn overall!